pytest.ini

```diff
@@ -1,3 +1,4 @@
 [pytest]
 filterwarnings =
     ignore:the imp module:DeprecationWarning
+    ignore:parse functions are required to provide a named argument:PendingDeprecationWarning
```

Comment:
This is weird since it seems like `setuptools_scm` is triggering its own pending deprecation. Or maybe I'm misunderstanding, but in any case it can be ignored. If it turns into a deprecation warning (no longer pending) then we'll see it again.